RadarSDK/RadarUser.h

```diff
@@ -141,10 +142,20 @@ typedef NS_ENUM(NSInteger, RadarLocationSource);
 @property (assign, nonatomic, readonly) RadarLocationSource source;
 
 /**
- A boolean indicating whether the user's IP address is a known proxy. May be `false` if Fraud is not enabled.
+ `true` if the user's IP address is a known proxy. May be `false` if Fraud is not enabled.
```

Comment:
I'd rather keep this as "A boolean indicating" for consistency with other fields on this object (ex. `stopped`, `foreground`)

Reply:
Done.